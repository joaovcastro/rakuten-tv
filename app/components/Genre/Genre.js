import React from 'react';
import PropTypes from 'prop-types';

const Genre = ({ name, icon }) => (
  <div className="genre-container" >
    <div className="genre">
      <img src={icon} />
    </div>
    <p> {name} </p>
  </div>
);

Genre.propTypes = {
  name: PropTypes.string,
  icon: PropTypes.string,
};

export default Genre;
