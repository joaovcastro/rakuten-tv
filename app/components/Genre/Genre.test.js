import React from 'react';
import { shallow } from 'enzyme';

import Genre from './Genre';

describe('<Genre />', () => {
  it('correctly renders', () => {
    const wrapper = shallow(<Genre name="genre" icon="url"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
