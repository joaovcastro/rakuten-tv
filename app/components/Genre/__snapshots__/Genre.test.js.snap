// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Genre /> correctly renders 1`] = `
<div
  className="genre-container"
>
  <div
    className="genre"
  >
    <img
      src="url"
    />
  </div>
  <p>
     
    genre
     
  </p>
</div>
`;
